.eslintrc.cjs

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+	"parserOptions": {
+		"sourceType": "module",
+	},
+	"env": {
+		"node": true,
+		"es2022": true
```

Comment:
top level await is handled by `es2022`. If we decide to go with `es2022` we should update also the other UI5 tooling repositories. @matz3 @RandomByte whats your opinion on that one?

Reply:
Sure, sounds good 👍

Reply:
PRs for all other repos are created